### PR TITLE
Add troubleshoot in ConfigV2Activity

### PR DESCRIPTION
**Pull request by ketal178**
# Add troubleshoot in ConfigV2Activity

## 更改内容 / Types of Changes
- 增强优化 Enhancement/optimization
- UI界面 User Interface

## 由这个PR解决或关闭的Issues / Issues Fixed or Closed by This PR

-  

## 检查列表 / Checklist

- [x] 我的代码符合本仓库的代码规范 My code follows the code style of this project
- [x] 我已经测试了这些更改，它们可以正常工作，既不会破坏原有任何功能(或我可以解决)，也不会影响对旧版本的支持 I have tested the changes and verified that they work and don't break anything(as well as I can manage) or drop the support for previous versions.
- [x] 我已合并了对后续工作无意义的commit并确认不会对后续维护造成困扰 I have merged commits that are meaningless for follow-up work and confirmed that they will not cause damage to follow-up maintenance

### CODE REVIEW

File: app/src/main/java/nil/nadph/qnotified/activity/TroubleshootActivity.java [hunk 1]
```diff
@@ -463,12 +464,16 @@ public void onClick(View v) {
                 @Override
                 public void onClick(DialogInterface dialog, int which) {
                     try {
+                        Long qq = Utils.getLongAccountUin();
+                        new File(
+                            HostInformationProviderKt.getHostInfo().getApplication().getFilesDir()
+                                .getAbsolutePath() + "/qnotified_" + qq + ".dat").delete();
```
Comment (NextAlone) [inline app/src/main/java/nil/nadph/qnotified/activity/TroubleshootActivity.java:470]: 有这个斜杠吗

File: app/src/main/java/nil/nadph/qnotified/activity/TroubleshootActivity.java [hunk 2]
```diff
@@ -488,9 +493,12 @@ public void onClick(DialogInterface dialog, int which) {
                 @Override
                 public void onClick(DialogInterface dialog, int which) {
                     try {
+                        new File(
+                            HostInformationProviderKt.getHostInfo().getApplication().getFilesDir()
+                                .getAbsolutePath() + "/qnotified_cache.dat").delete();
```
Comment (NextAlone) [inline app/src/main/java/nil/nadph/qnotified/activity/TroubleshootActivity.java:498]: 同上

File: app/src/main/java/nil/nadph/qnotified/activity/TroubleshootActivity.java [hunk 3]
```diff
@@ -514,9 +522,12 @@ public void onClick(View v) {
                 CustomDialog dialog = CustomDialog.create(TroubleshootActivity.this);
                 dialog.setPositiveButton("确认", (dialog1, which) -> {
                     try {
+                        new File(
+                            HostInformationProviderKt.getHostInfo().getApplication().getFilesDir()
+                                .getAbsolutePath() + "/qnotified_config.dat").delete();
```
Comment (NextAlone) [inline app/src/main/java/nil/nadph/qnotified/activity/TroubleshootActivity.java:527]: 同上

**Comment by ketal178** [comment]
我复制粘贴的还会错------------------&nbsp;原始邮件&nbsp;------------------
***@***.***&gt;
发送时间:&nbsp;2021年4月13日(星期二) 上午10:27
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [ferredoxin/QNotified] Add troubleshoot in ConfigV2Activity (#817)

**Review by NextAlone — APPROVED** [review]

**Review by cinit — APPROVED** [review]

**Pull request merged**